code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -135,7 +135,9 @@
 
 // Modifiers for work chance
 /mob/living/simple_animal/hostile/abnormality/proc/work_chance(mob/living/carbon/human/user, chance)
-	return chance
+	if (user.physiology.work_success_mod <= 0)
```

Comment:
There is no need to put it here at all.

Reply:
What do you mean by that? Are you referring to the check for if it's less than or equal to 0? If so, I implemented that to prevent situations such as someone having such a terrible global work mod from something such as stacking debuffs that they get negative and work on an abnormality with a negative work rate (Such as Helper with Insight) and manage to turn it positive

Reply:
No, I am saying that it should be put in the abnormality datum check, not here.

Reply:
Moved to the datum check 👍